export_connectivity.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd `dirname $0`
```

Comment:
`$(command)` syntax is preferred to use of backticks. This appears to be a widely preferred convention.

https://github.com/azet/community_bash_style_guide#never-use-backticks